spyder/plugins/onlinehelp/tests/test_onlinehelp_config_dialog.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# ----------------------------------------------------------------------------
+# Copyright © Spyder Project Contributors
+#
+# Licensed under the terms of the MIT License
+# ----------------------------------------------------------------------------
+"""Tests for plugin config dialog."""
+
+try:
+    from unittest.mock import Mock
+except ImportError:
+    from mock import Mock  # Python 2
+
+# Test library imports
+import pytest
+
+# Local imports
+from spyder.plugins.onlinehelp.plugin import OnlineHelp
+from spyder.preferences.tests.conftest import config_dialog
+
+
+class MainWindowMock:
+    register_shortcut = Mock()
+
+
+@pytest.mark.parametrize(
+    'config_dialog',
+    # [[MainWindowMock, [ConfigPlugins], [Plugins]]]
+    [[MainWindowMock, [], [OnlineHelp]]],
+    indirect=True)
+def test_config_dialog(config_dialog):
+    configpage = config_dialog.get_page()
+    assert configpage is None
```

Comment:
Do we really need to add tests for a non-existing config pages? I'd say this is really not necessary and it could be confusing to new contributors.

Reply:
@ccordoba12 the idea is that if at some point we do add a config page for those plugins test would fail, which require the user to pay attention that tests need to be added to this plugin.

Reply:
Ok, I understand. However, I'm a bit -1 on this. Tests should be added when the confpage is added too.

So I'd say we should remove these tests for now.

Reply:
Ok will remove them!

Also I have a problem with the `help` plugin. I will post in a comment

Reply:
Ok, thanks!